docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -249,7 +248,7 @@ Versioning and deprecation
   * You can only remove things or make breaking changes in major releases.
 * We recommend that you ensure that if a deprecation is added in a collection version that is included in Ansible 3.y.z, the removal itself will only happen in a collection version included in Ansible 5.0.0 or later, but not in a collection version included in Ansible 4.0.0.
 * Content moved from ansible/ansible that was scheduled for removal in 2.11 or later MUST NOT be removed in the current major release available when ansible 2.10.0 is released. Otherwise it would already be removed in 2.10, unexpectedly for users! Deprecation cycles can be shortened (since they are now uncoupled from ansible or ansible-base versions), but existing ones must not be unexpectedly terminated.
-* We recommend you announce your policy of releasing, versioning and deprecation to contributors and users in some way. For an example of how to do this, see `the announcement in community.general <https://github.com/ansible-collections/community.general/issues/582>`_. You could also do this in the README.
+* The collection policy of releasing, versioning and deprecation MUST be announced to contributors and users in some way. For an example of how to do this, see `the announcement in community.general <https://github.com/ansible-collections/community.general/issues/582>`_. You could also do this in the README.
```

Comment:
This looks like a change from (implied) SHOULD to MUST, do we need to discuss it? I might also recommend we use the verb "publish" instead of "announce" since what this is really recommending/requiring is that the policy is published (available) and not that it's announced (sent/pushed as an event).

Reply:
the wording you're suggesting SGTM
also i thought it was a must because it looks like a must according to https://github.com/ansible-collections/overview/blob/main/collection_checklist.md :) IMO it makes sense to require it but, sure, we'll discuss this. Anyway we'll vote on this overhaul i think

Reply:
Oh, now i think that we shouldn't introduce new requirements.. i've made it a should. we should update the checklist to prevent requiring shoulds as musts in future:)